0.10/Dockerfile.rhel7

```diff
@@ -32,6 +35,8 @@ COPY ./s2i/bin/ $STI_SCRIPTS_PATH
 # run and build the applications.
 COPY ./contrib/ /opt/app-root
 
+RUN $PROMPT_FILE && npm install -g nodemon
```

Comment:
Ah... I'm not certain what to do in this circumstance. I'm not certain that such a package exists - where could I find this out? If it doesn't exist, what does creating a new supported RHEL package involve?